modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -167,7 +167,7 @@ const get = function (dom: DOMUtils) {
         return NodeType.isElement(node) && node.hasAttribute('href');
       },
 
-      onformat(elm, _fmt, vars) {
+      onformat(elm, _fmt, vars: FormatVars | undefined) {
```

Comment:
```suggestion
      onformat(elm, _fmt, vars?: FormatVars) {
```
Do we want something like this, or do we actually expect people to pass two arguments just with one of them maybe being undefined?

Reply:
Good catch, I'll fix this.